docs/projects/ad5766_sdz/index.rst

```diff
@@ -0,0 +1,269 @@
+.. _ad5766_sdz:
+
+AD5766-SDZ HDL project
+===============================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`AD5766`/ :adi:`AD5767` are 16-channel, 16-/12-bit, voltage output
+Digital-to-Analog Converters (DAC). The DAC generates output voltage ranges from
+an external 2.5 V reference. Depending on the span selected, the mid-point of
+the output span can be adjusted allowing for a minimum output voltage as low as
+−20 V or a maximum output voltage of up to +14 V.
+The :adi:`AD5766`/ :adi:`AD5767` have integrated output buffers which can sink
```

Comment:
Add a blank line between paragraphs to break them in HTML output.
Comment applies to all paragraphs in this section.

Reply:
Done.